syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -292,6 +296,14 @@ def handle_func_command(cls, command):
             if args_type not in FrameworkTensor:
                 return args_type.handle_func_command(command)
 
+            # Check that the function is not overloaded
+            try:
+                # Try to get recursively the attributes in cmd = "<attr1>.<attr2>.<attr3>..."
+                command = cls.rgetattr(cls, cmd)
+                return command(*args, **kwargs)
+            except AttributeError:
+                pass
+
```

Comment:
I think you can remove the occurence of the same code below at line 316.

Reply:
Hey @LaRiffle, the code above is important because else it doesn't handle case where we pass shift parameter as a tensor.
For example,
```python
x = torch.tensor([1.0, 2.0, 3, 4, 5])
index = torch.tensor([-1.0])
result = torch.roll(x, index)
```
exits with
```python
TypeError: roll(): argument 'shifts' (position 2) must be tuple of ints, not Tensor
```
Do we have any alternative or this is fine?

Reply:
I think you should put that block out of the 
```
<here>
try:
...
except PureFrameworkTensorFoundError
```
it should work better!

Reply:
This way you can indeed remove the duplicate and only keep a single occurence

Reply:
Hey,
That too results in an error 
```python 
RuntimeError: Error, Please consider calling ".get" method instead of ".item" method, so you can be safely getting the item you need.
```
when we pass syft tensor (wrapper) as shift parameter.

This might be because the ```handle_command_function``` calls correct overloaded function but arguments needs to be resolved first (wrapper to be converted to the framework tensor) before passing to the overloaded function.